### ISSUE TITLE
My ads won't run on IOS

### ISSUE THREAD

**Issue by Felipe634** [kloder-games/godot-admob#158]
**OS target (Android/iOS):**
IOS

**Godot version:**
3.2.3

**Issue description:**

I am releasing my second game and I have used this admob plugin for my first game and the ads started working after 6 days of being live in the app store. This time it has been 2 weeks and still no ads are working but I did nothing different from the first game. Any idea on how to fix this?

**Comment by gumaciel** [comment]
Hello @Felipe634 i recommend you to use AdMob with this repository: https://github.com/Poing-Studios/Godot-AdMob-Android-iOS

**Comment by Felipe634** [comment]
Is that because the other one does not work anymore?

**Comment by gumaciel** [comment]
No, this repository (https://github.com/kloder-games/godot-admob) works on all Godot Versions for iOS for 3.0 to 3.2.3, and for Android on 3.0 to 3.2.1, **BUT** it's deprecated and nobody does more updates.

This repository: https://github.com/Shin-NiL/Godot-Android-Admob-Plugin, works fine just on Godot Versions for Android 3.2.2 to 3.2.3 (future will work on 3.2.4)

And this repository (which i recommend): https://github.com/Poing-Studios/Godot-AdMob-Android-iOS works on all Godot 3 versions for iOS and 3.2.2+ for Android, and will soon include the iOS Plugins for 3.2.4 like the Android Plugins: https://github.com/Poing-Studios/Godot-AdMob-Android-iOS/issues/27

Actually exists other repositories but i don't recommend at all because don't have all ads formats, configurations, etc, i just recommend these three: kloder, shin-nil, and poing

Other thing about the kloder repository: With the new versions of Google Mobile Ads SDK for Android and iOS, need to have a update on the source code of AdMob, witch is currently in progress on Poing: https://github.com/Poing-Studios/Godot-AdMob-Android-iOS/issues/37, so if you try to use the newest version of AdMob SDK the App will crash

**Comment by Felipe634** [comment]
So I should use  https://github.com/Poing-Studios/Godot-AdMob-Android-iOS   ?

**Comment by gumaciel** [comment]
Yes, i recommend you to use this one, it already have compiled files (https://github.com/Poing-Studios/Godot-AdMob-Android-iOS/releases) so you don't need to compile by your self, just follow the [instructions](https://github.com/Poing-Studios/Godot-AdMob-Android-iOS#ios-v300) (very similar to kloder), and will work fine on all version supported


And also, pay attention for this (it's not mandatory, but is good to configure): https://github.com/Poing-Studios/Godot-AdMob-Android-iOS#user-messaging-platform-ump

**Comment by Felipe634** [comment]
Thank you!

**Comment by Felipe634** [comment]
Do you know if there is a video to help me do the code or is the code the same as kloder?

**Comment by gumaciel** [comment]
@Felipe634 currently doesn't have a video tutorial, just the README, please try to run with this example project: https://github.com/Poing-Studios/Godot-AdMob-Android-iOS/tree/master/example

What you need to do:

1 - Copy https://github.com/Poing-Studios/Godot-AdMob-Android-iOS/tree/master/admob_api inside your game project and add "MobileAds.tscn" as Singleton and AutoLoad on ```Project Settings```
2 - Do these things if you want to work with Android: https://github.com/Poing-Studios/Godot-AdMob-Android-iOS#android-v322
3 - iOS: https://github.com/Poing-Studios/Godot-AdMob-Android-iOS#ios-v300

And please, try with the example project to see how AdMob works

**Comment by gumaciel** [comment]
@Felipe634 did you manage to work?
